drivers/driver.cmake

```diff
@@ -0,0 +1,63 @@
+
+if(CMAKE_COMPILER_IS_GNUCC)
+    set(LIBS_C_FLAGS -Wmissing-declarations -Wmissing-prototypes)
+endif(CMAKE_COMPILER_IS_GNUCC)
+
+if(CMAKE_COMPILER_IS_CLANG)
+    set(LIBS_C_FLAGS -Wmissing-declarations -Wmissing-prototypes -Wdocumentation -Wno-documentation-deprecated-sync -Wunreachable-code)
+endif(CMAKE_COMPILER_IS_CLANG)
+
+if(CMAKE_COMPILER_IS_MSVC)
+    option(MSVC_STATIC_RUNTIME "Build the libraries with /MT compiler flag" OFF)
```

Comment:
What's the scope of an `option`? If it's global (which I guess it would be since it can be set on the command line when you run `cmake`, how does it work if `driver.cmake` is included by multiple drivers?